.gitlab-ci.yml

```diff
@@ -256,19 +256,25 @@ deploy-prod-stable:
 # Notify
 ########################################################################################################################
 
-include: 'https://gitlab-templates.ddbuild.io/slack-notifier/v1/template.yml'
-
-notify-feature-branch-failure:
-  extends:
-    - .slack-notifier.on-failure
-    - .feature-branches
-
 .prepare_notification:
-  extends: .slack-notifier-base
+  stage: notify
+  extends:
+    - .base-configuration
   before_script:
     - COMMIT_MESSAGE=`git show-branch --no-name HEAD`
     - BUILD_URL="$CI_PROJECT_URL/pipelines/$CI_PIPELINE_ID"
     - COMMIT_URL="$CI_PROJECT_URL/commits/$CI_COMMIT_SHA"
+    # If the pipeline was launched by a user, send pings to them (eg, creating a pipe on master without commiting)
+    - if [[ $GITLAB_USER_LOGIN = "gitsync" ]]; then AUTHOR_EMAIL=$(git show -s --format="%ae" HEAD); else AUTHOR_EMAIL=$GITLAB_USER_EMAIL; fi
+
+notify-feature-branch-failure:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This is great, loving it :) could we do the same on success too?

Reply:
Sure, that could be nice ;)